modules/uhttp/client/client_filters_test.go

```diff
@@ -74,7 +74,7 @@ func TestExecutionChainFiltersError(t *testing.T) {
 }
 
 func withOpentracingSetup(t *testing.T, registerFunc auth.RegisterFunc, fn func(tracer opentracing.Tracer)) {
-	tracer, closer, err := tracing.InitGlobalTracer(&jconfig.Configuration{}, "Test", ulog.NopLogger, metrics.NopCachedStatsReporter)
+	tracer, closer, err := tracing.InitGlobalTracer(&jconfig.Configuration{}, "Test", zap.New(nil), metrics.NopCachedStatsReporter)
```

Comment:
all these `zap.New(nil)` calls are starting to stand out as a smell. Any way we can rework it so that a ZapCore is a variadic option and you scan for it first? Just a general comment any time I see `nil` as a required param repeatedly.